components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesStorage.kt

```diff
@@ -27,7 +27,7 @@ open class PlacesStorage(context: Context) : Storage, SyncableStore {
     private val storageDir by lazy { context.filesDir }
 
     @VisibleForTesting
-    internal open val places: Connection by lazy {
+    open val places: Connection by lazy {
```

Comment:
I'm not sure if I should do this but I can't override in my test otherwise IIUC?